readablepdf

```diff
@@ -354,9 +358,20 @@ while true; do
 	esac
 done
 
+
+echo "Estimating CPU-Threads"
+# Estimate CPU Cores (see pdfsandwich code for reference)
+CPU_THREADS=$(cat /proc/cpuinfo | grep processor | awk '{a++} END {print a}')
+OMP_THREAD_LIMIT=$CPU_THREADS
+MAX_JOBS=$(expr $CPU_THREADS / 4)
+# tesseract only uses 4 threads
```

Comment:
I'll take a look at performance on my system later, but this assumption that Tesseract will always use a maximum of 4 threads seems guaranteed to come back to bite. ;-)

Reply:
Tesseract 4 does at the moment, at least according to the docs and to observing my system resources. How this changes in further development (as far as I can tell, there are changes planned for multithreading), is of course not foreseeable.
But yes – from a standpoint of longevity it's probably stupid.